src/components/Support/SupportNavItem.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from "../shared/Button/Button";
+
+interface Props {
+  label: string;
+  icon: JSX.Element;
+  href: string;
+}
+
+export const SupportNavItem = ({ label, icon, href }: Props) => (
```

Comment:
Tbh could be inlined in mapping in the `SupportNav` but I like such small components 👍 
So we can leave it like that 😉